app/views/submissions/provenance.html.erb

```diff
@@ -101,7 +101,7 @@
 
       <% labwares.each_with_index do |labware, i| %>
 
-        <div role="tabpanel" class="tab-pane <%= 'active' if i==0 %>" id="Labware<%= labware.labware_index %>">
+        <div  style="overflow: auto;" role="tabpanel" class="tab-pane <%= 'active' if i==0 %>" id="Labware<%= labware.labware_index %>">
```

Comment:
Please could you revert this change? My latest changes on devel add this to a div containing just the table, so with this change as well we'd have a scrolling table inside a scrolling panel